management/templates/system-backup.html

```diff
@@ -235,6 +261,13 @@ <h3>Available backups</h3>
           var host = hostpath.shift();
           $("#backup-target-s3-host").val(host);
           $("#backup-target-s3-path").val(hostpath.join('/'));
+        } else if (r.target.substring(0, 5) == "b2://") {
+          $("#backup-target-type").val("b2");
+          var hostpath = r.target.substring(5).split('@');
+          var usernamepass = hostpath.shift().split(":");
+          $("#backup-target-rsync-b2-user").val(usernamepass[0]);
+          $("#backup-target-rsync-b2-user").val(usernamepass[1]);
```

Comment:
There's a bug here setting: `backup-target-rsync-b2-user` twice. I guess it should be a password variable of some sort.
```
$("#backup-target-rsync-b2-user").val(usernamepass[1]);
$("#backup-target-rsync-b2-PASSWORD???").val(usernamepass[1]);
```